### DISCUSSION TITLE
Suggest adding an option to hide the notification icons in the bottom right corner of the taskbar in Windows 11 style taskbar .

### DISCUSSION THREAD

**Discussion by ShawnO28** [valinet/ExplorerPatcher#3784] [Ideas]
Suggest adding an option to hide the notification icons in the bottom right corner of the taskbar in Windows 11 style taskbar .

**Comment by pyrates999** [comment]
Windows 10 and 11 already does this.  Right click on the notification icon on the bottom right and set focus assist to alarms only or priority only.  If you wish to see notifications again, do the same thing but set focus assist to off.

**Reply by ShawnO28** [reply]
Thank you.
I am referring to the notification icon that permanently resides in the taskbar in Windows 11 style (which is displayed even when there are no notifications). In earlier versions of Windows 11, you could hide this permanent notification icon by disabling the notification feature. However, in later versions, even after disabling notifications, the permanent notification icon still appears in the taskbar (it shows even when there are no notifications). Modifying the registry to hide the icon prevents the date and time click from showing the calendar card. I hope there can be an option to hide the notification icon (which shows regardless of whether there are notifications) while still allowing the date and time click to show the date and time card (similar to the Windows 10/11 style calendar card). Thank you.
![image](https://github.com/user-attachments/assets/e5cf2a04-645c-440d-a685-a0d18fd5e575)

**Comment by pyrates999** [comment]
does doing this work? https://www.thewindowsclub.com/disable-notification-center-in-windows-11

**Reply by ShawnO28** [reply]
Thank you. After trying the methods in the article, disabling the notification center in Windows 11 caused the calendar card to be unable to open. After adding the registry entry to hide the notification bell icon and restarting, the notification bell icon still appeared (Windows 11 23H2). In earlier versions of Windows 11, it was possible to hide the notification icon while still being able to click on the date in the taskbar to show the calendar card, but this is not possible in the 23H2 version.
The following PDF and DOCX files are attachments of the relevant screenshots.
[file.pdf](https://github.com/user-attachments/files/17341692/file.pdf)

**Reply by pyrates999** [reply]
Please upload it as each image.  I'm not opening a random pdf.

**Reply by ShawnO28** [reply]
![Screenshot (54)](https://github.com/user-attachments/assets/f341ba8e-ba45-4297-9b8b-89d18385ad50)

![Screenshot (55)](https://github.com/user-attachments/assets/4c0853fe-a9d8-4b01-af01-59cbfae3471c)

![Screenshot (56)](https://github.com/user-attachments/assets/1991582a-d463-47f3-8f8d-05cb6a9d09e7)

![Screenshot (57)](https://github.com/user-attachments/assets/8f3b2234-88d4-4e7a-9fa3-a5b87a8ebc1e)

![Screenshot (58)](https://github.com/user-attachments/assets/cc0785a3-e1f0-4bab-ad5a-6e8e70f4e316)

**Reply by pyrates999** [reply]
Thanks.  We'll see if it gets implemented into EP then.
